### PR TITLE
Fixing up the arm subsystem versions

### PR DESCRIPTION
**Pull request by Petermarcu**
https://github.com/dotnet/coreclr/issues/8238

### CODE REVIEW

**Comment by Petermarcu** [comment]
@gkhanna79

File: CMakeLists.txt [hunk 1]
```diff
@@ -293,10 +293,13 @@ if (WIN32)
   #
   # Disable the following line for UNIX altjit on Windows
   set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /MANIFEST:NO") #Do not create Side-by-Side Assembly Manifest
-  if (CLR_CMAKE_HOST_ARCH STREQUAL arm)
-    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /SUBSYSTEM:WINDOWS,6.02") #windows subsystem - arm minimum is 6.02
+
+  if (CLR_CMAKE_PLATFORM_ARCH_ARM)
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /SUBSYSTEM:WINDOWS,6.02") #windows subsystem - arm minimum is 6.02  
+  elseif(CLR_CMAKE_PLATFORM_ARCH_ARM64)
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /SUBSYSTEM:WINDOWS,6.03") #windows subsystem - arm64 minimum is 6.02
```
Comment (jkotas) [inline CMakeLists.txt]: Nit - comment says 6.02, code has 6.03

**Comment by Petermarcu** [comment]
@dotnet-bot test Windows_NT x86 Checked Build and Test please

File: CMakeLists.txt (hunk 1 above)
Comment (Petermarcu) [inline CMakeLists.txt]: Thanks. I'll fix that.

**Pull request merged**